routes/root.js

```diff
@@ -40,12 +40,20 @@ route.get('/logout', (req, res) => {
 })
 
 route.get('/leaderboard/:year?', (req, res) => {
-  const { year } = req.params
-  const options = {
-    page: req.query.page || 1,
-    size: req.query.size || config.PAGINATION_SIZE,
-    year
-  }
+    let { year } = req.params
+    const validYears = [2020, 2019, 2018]
```

Comment:
This should be an array of strings. Not working with integers :)

Reply:
Can you create a PR for it?